code/kit_version.R

```diff
@@ -0,0 +1,107 @@
+# SET UP
+setwd("C:/Users/kitsc/KIT_code_testing/data")
+library(haven)
+library(tidyverse)
+library(plm)
+library(dplyr)
+
+# DEFINITIONS
+
+##	seed
+	seed=51
```

Comment:
Personally I only indent things that are nested within something else (for example, a loop, a function call or definition, etc.). As with all coding style points, the most important thing is consistency, but following similar styles to other people's can also be helpful. Check our [Hadley Wickham's style guide](http://adv-r.had.co.nz/Style.html), for example.